client/src/components/App/AppToolbar/TopToolbar/TopToolbar.tsx

```diff
@@ -2,21 +2,71 @@ import React from 'react';
 import {useSelector} from 'react-redux';
 import {Toolbar} from '@material-ui/core';
 import {Typography} from '@material-ui/core';
+import Tooltip from '@material-ui/core/Tooltip';
+import { ToggleButtonGroup, ToggleButton } from '@material-ui/lab';
 
-import useStyles from './TopToolbarStyles';
+import User from 'models/User';
 import StoreStateType from 'redux/storeStateType';
 
+import useStyles from './TopToolbarStyles';
+import useTopToolbar from './useTopToolbar';
+
+const toggleMessage ='מה הסטטוס שלך?';
+
 const TopToolbar: React.FC = (): JSX.Element => {
     const classes = useStyles({});
-    const userName = useSelector<StoreStateType, string>(state => state.user.name);
-  
+    const firstUserUpdate = React.useRef(true);
+
+    const user = useSelector<StoreStateType, User>(state => state.user);
+
+    const [isActive, setIsActive] = React.useState<boolean>(false);
+
+    const {getUserActivityStatus, setUserActivityStatus} = useTopToolbar({ setIsActive });
+   
+    const activeButtonStyle = {
+        backgroundColor: '#117243',
+        color: 'white'
+    }
+
+    const notActiveButtonStyle = {
+        backgroundColor: '#9e2a2b',
+        color: 'white'
+    }
+
+    React.useEffect( () => {
+        if (firstUserUpdate.current) {
+            firstUserUpdate.current = false;
+            return;
+        } else {
+            getUserActivityStatus();
+        }
+    }, [user])
+
     return (
-        <Toolbar className={classes.toolbar}>
+        <Toolbar className={classes.toolbar} >
             <div className={classes.rightToolbarSection}>
                 <img alt='logo' className={classes.logo} src='./assets/img/logo.png'></img>
                 <Typography variant='h4' className={classes.centering}><div className={classes.systemName}>אבן יסוד</div></Typography>
             </div>
-            <Typography>שלום, {userName}</Typography>
+            <div className={classes.leftToolbarSection}>
+                <Tooltip title={toggleMessage} arrow>
+                    <ToggleButtonGroup value={isActive} exclusive className={classes.isActiveToggle}>
+                        <ToggleButton className={classes.toggle} 
+                            style={isActive ? activeButtonStyle : {}}
```

Comment:
why style and not class? same for line 61

Reply:
This is how the common toggle implemented too.. Class doesn't work here. I would like to know how do you suggest to do it.

Reply:
a classes sub-object returns a string you could add it conditionally 🤷‍♀️

Reply:
I tried. Didn't work for me. If you did it I would like to see how you made it happen.

Reply:
```suggestion
                            className={`${classes.toggle} ${isActive ? classes.activeButtonStyle:''}`} 
```

Reply:
```suggestion
							const toggleClasses = isActive :[classes.toggle, classes.activeButtonStyle ] : [classes.toggle] 
                            className={toggleClasses.join(' ')} 
```

Reply:
there are probably better ways than these but those give out a general idea

Reply:
Doesn't work both ways.